x-pack/legacy/plugins/siem/public/components/events_viewer/events_viewer.test.tsx

```diff
@@ -25,11 +27,21 @@ mockUseKibanaCore.mockImplementation(() => ({
   uiSettings: mockUiSettings,
 }));
 
+const mockUseFetchIndexPatterns: jest.Mock = useFetchIndexPatterns as jest.Mock;
+jest.mock('../../containers/detection_engine/rules/fetch_index_patterns');
+mockUseFetchIndexPatterns.mockImplementation(() => [
+  {
+    browserFields: mockBrowserFields,
+    indexPatterns: mockIndexPattern,
+  },
+]);
+
 const from = 1566943856794;
 const to = 1566857456791;
 
 describe('EventsViewer', () => {
-  test('it renders the "Showing..." subtitle with the expected event count', async () => {
+  // TODO Fix test
```

Comment:
@XavierM -- the `mockEventViewerResponse` isn't being used anymore with our change from `WithSource` to `useFetchIndexPatterns`. Thoughts on what we can change in the mock/provider to get this working again?

Reply:
Docs got our back: https://www.apollographql.com/docs/react/development-testing/testing/#mockedprovider

> Your mock request's variables object must exactly match the query variables sent from your component.

Was a mismatch in the variables object -- updating it fixes this issue 🙂